### DISCUSSION TITLE
Misc Feedback from first user.

### DISCUSSION THREAD

**Discussion by Carreau** [pretalx/pretalx#1430] [Ideas] [answered]
Hello, 

I recently had to use pretalx for organizing a conference, and I know how it car be hard to collect first-time user feedback so I'm trying to do so while it is still fresh in my mind. 

Note that I did not deploy the software and didn't figure out how to find the version number, so it may be my feedback has likely been already fixed. 

- I found that timezone should be explicit in more places. Setting the deadline of CFP I was unsure which timezone it was using, and I don't think this is shown to users either.

- I appears that "mail" > "Compose Emails" does not let you select proposals to which send emails by tags. I think that would be tremendously helpful.

- In this same screen, I would recommend to change "Send" to something like "Send to outbox", to make it clearer the mails won't be directly send.

- I did not found a way to edit the rejection template for proposal, I think it would be useful. 

- I would be really good to have the ability to select the "Accept Template" among other template. We had multiple accept mails depending on some criteria, and copy-pasting the accept mails template instead of being able to say "this proposal should use this accept/reject template mail", was a long and tedious process.

- I think it might be helpful to allow an option when one proposal has multiple speakers to send one email to all the speakers instead of one per speaker so that it creates a thread. But that's a minor point.

I'm now starting on the "schedule" tab.
 
- I'd love a way to change the vertical scale, it's quite hard to see/set 15 minutes slots, but I don't need to see a full day vertically. 

- Some of my colleagues are confused by the time on the side. They are confused to have the number in the middle of the case and I thing that the current layout could be improved by placing the time next to the ticks instead of in between;Pardon my poor css skills:

```
.timestep-box{
    border: none;
}

.timestep-box > span:after{
    content: '—';
    
}
```

<img width="116" alt="Screen Shot 2023-02-18 at 11 34 00" src="https://user-images.githubusercontent.com/335567/219855726-4e598b58-2151-44cb-949d-106c19cceab5.png">

Thanks again,

**Comment by rixx** [comment, marked as answer]
Thank you for your feedback! Just for future reference / for people stumbling into this:

- Timezone has been made more explicit in some places – not in the settings, but in availability forms and the schedule editor. Timezone is also always shown in the schedule and in user-facing schedule information.
- "Send to outbox" is a great phrasing suggestion, thanks! Added in f7f294f2
- The rejection template can definitely be edited – it was missing for a short while, which was fixed in December in 8b0421c0
- I'm afraid multiple accept templates are currently out of scope for pretalx – people would want multiple criteria (e.g. by track, tag(s), proposal type, language), and then there would be a lot of confusion over which template would have precedence. I'd be open to a feature request and/or sponsored development, but otherwise wouldn't prioritise this on my own.
- More involved email handling will also remain out of scope for now.
- I'd agree that the current pretalx schedule editor is terrible to the point of being entirely useless, and I actively feel bad for everybody who has to use it. I'm sorry :( In case it helps, here's a preview of the new schedule editor that is in the works (and sadly has been, for a while, as it involves changing our entire build system, translations, etc):


https://user-images.githubusercontent.com/2657547/219954650-9b99aebc-4678-4d16-9fa9-1b69174498c5.mp4

**Comment by Carreau** [comment]
Thanks for the reply, as I said it's my first time using the software, but if we use it more often and feel the needs, we can either try to find some funds to get those features implemented, or work toward contributing them (or at least helping you with some maintenance) .

The new schedule management looks great !

**Reply by rixx** [reply]
Definitely happy to talk about contributions or funding, but also again: thank you for the feedback – like you said initially, getting first impression is hard, and I appreciate it a lot!
